### PR TITLE
Generic Badges

### PR DESCRIPTION
**Pull request by braxtonCoats**
## PR Checklist

Check these before submitting your pull request:

- [x] Visual and behavioral components are separated
- [x] Exported components are documented with examples
- [x] Props have JSDoc comments
- [x] All relevant visual sub-components can be overridden via props
- [x] Any extra props are spread into the outermost rendered element

## Breaking Changes

* Describe all breaking changes here

## Changes to Existing Components

Moved `NewBadge` to live under the more general `Badge` file, didnt remove `NewBadge` to avoid breaking features.

## New Visual Components

Created generic tags.

Screen Shot 2019-04-09 at 11 43 47 AM

### CODE REVIEW

File: src/components/Badge/Badge.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+`;
+
+Badge.Primary = styled(Badge)`
+  background: ${get('colors.primary.default')};
+  color: ${get('colors.text.inverted')};
+`;
+
+Badge.Accent_0 = styled(Badge)`
```
Comment (a-type) [inline src/components/Badge/Badge.js]: PascalCase is our preference for all naming:

```suggestion
Badge.Accent0 = styled(Badge)`
```

File: src/components/Badge/Badge.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  background: ${get('colors.accents-0-default')};
+  color: ${get('colors.text.inverted')};
+`;
+
+Badge.Accent_1 = styled(Badge)`
```
Comment (a-type) [inline src/components/Badge/Badge.js]:
```suggestion
Badge.Accent1 = styled(Badge)`
```

File: src/components/Badge/Badge.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Badges are used to show which product verticals are applicable to any given section in the Bandwidth Dashboard.
+
+New Badge
+```javascript
+<Badge.New disabled />
```
Comment (a-type) [inline src/components/Badge/Badge.md]: Is `disabled` meaningful?

Comment (braxtonCoats) [reply src/components/Badge/Badge.md]: Nope, this was a mistake.

**Comment by braxtonCoats** [comment]
Updated NewBadge to use Badge.New

File: src/components/NewBadge/NewBadge.js [hunk 4]
```diff
@@ -1,20 +1,9 @@
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
```
Comment (a-type) [inline src/components/NewBadge/NewBadge.js]: These two imports are unused now and can be deleted.

**Comment by braxtonCoats** [comment]
Updated NewBadge to remove unneeded imports.

**Comment by zelitzsi** [comment]
@braxtonCoats If this is ready to go, we can merge it in for you.

**Comment by braxtonCoats** [comment]
it is good to go @zelitzsi

**Pull request merged**